### PR TITLE
Add Qt version of Pelux

### PR DESCRIPTION
**Pull request by DunderRoffe**

### CODE REVIEW

File: pelux-qt.xml [hunk 1]
```diff
@@ -5,6 +5,9 @@
   <remote fetch="git://code.qt.io/"             name="qt-project" />
 
   <!-- Qt and Appman -->
-  <project remote="qt-project" revision="3425517323dbb47641541e9905821f3e3633be5e" name="yocto/meta-boot2qt" path="sources/meta-boot2qt"/>
+  <project remote="qt-project"
+           revision="dev"
```
Comment (GordanM) [inline pelux-qt.xml]: The revision used and maintained currently is krogoth. I would recommend pointing to it.

**Comment by DunderRoffe** [comment]
@GordanM I get errors when fetching qtwayland from github. Do you know anything about this?

```
ERROR: qtwayland-5.7.1+gitAUTOINC+7d527c1914-r0 do_fetch: Fetcher failure: Unable to find revision 7d527c1914710a511c06713bbda1572b767cea34 in branch 5.7 even from upstream
ERROR: qtwayland-5.7.1+gitAUTOINC+7d527c1914-r0 do_fetch: Function failed: Fetcher failure for URL: 'git://github.com/qtproject/qtwayland.git;name=qtwayland;branch=5.7;protocol=git'. Unable to fetch URL from any source.
ERROR: Logfile of failure stored in: /mnt/storage/visj/pelux-qt2/build/tmp/work/corei7-64-bistro-linux/qtwayland/5.7.1+gitAUTOINC+7d527c1914-r0/temp/log.do_fetch.24617
ERROR: Task 3412 (/mnt/storage/visj/pelux-qt2/sources/meta-qt5/recipes-qt/qt5/qtwayland_git.bb, do_fetch) failed with exit code '1'
```

**Comment by GordanM** [comment]
@DunderRoffe Samuli did a good thing and updated the QtApplicationManager and the proper QtWayland revision to go with it (http://code.qt.io/cgit/yocto/meta-boot2qt.git/commit/?h=krogoth&id=09f8c75cfeb0ea6d6b77dbd25cfbf779aa395b45). 

The bitbake error points that the QtWayland revision in question is not found in the 5.7 branch.

The latest version of QtAS components dependent on the 5.8 meta-qt5 branch. Like I suggested previously it would be wiser to track that branch/revision (check the top commit of http://code.qt.io/cgit/yocto/meta-qt5.git/log/?h=5.8) instead of pointing to morty on mainline meta-qt5.

**Comment by DunderRoffe** [comment]
@GordanM ok did not realize that this would impact this. Trying to change that and build locally. Thanks for the tip

**Comment by DunderRoffe** [comment]
Seems like it worked. So is my assumption that 5.7 is broken correct then? Since I is not possible to download some of the gits?

**Comment by DunderRoffe** [comment]
Actually it might be better to get your pr merged instead of me doing the same thing @GordanM.

**Comment by GordanM** [comment]
Thanks @DunderRoffe! I will merge my PR and you can continue for there on. 

Regarding 5.7, it's not broken as such but the integration with certain QtAS components (mainly QtAppMan) is far from ideal. Also, the meta-boot2qt layer's krogoth branch relies on Qt 5.8. Basically, there should be no problem using QtAppMan with Qt 5.7 but then you would need to discard the meta-boot2qt layer and that would be unfavorable.

**Comment by DunderRoffe** [comment]
Rebased on your commit and both the qt and non-qt images build on Jenkins. I feel that this is ready for merge. @GordanM and @socec are there anything you would like me to add :)

**Comment by socec** [comment]
I'm good :)

**Pull request merged**